app/forms.py

```diff
@@ -1,5 +1,5 @@
 from flask_wtf import FlaskForm
-from wtforms import StringField, HiddenField, TextAreaField, IntegerField, SubmitField, FloatField, SelectField, validators, PasswordField, Form, FileField
+from wtforms import DecimalField, StringField, HiddenField, TextAreaField, IntegerField, SubmitField, FloatField, SelectField, validators, PasswordField, Form, FileField
```

Comment:
@zhanerken validators has been imported twice. please import it once

Reply:
@imran072 i found an error when deleting the listing, fixed it in a new push, now it has cascade deletion of both (offer + listing)